rules/xcodeproj.bzl

```diff
@@ -425,6 +425,7 @@ def _populate_xcodeproj_targets_and_schemes(ctx, targets, src_dot_dots):
         compiled_sources = [{
             "path": paths.join(src_dot_dots, s.short_path),
             "optional": True,
+            "buildPhase": "none",
```

Comment:
Have u tested whether files not inside compiled source phase will be indexed by XCode? I am pretty they are added to the compiled source list is so that XCode knows them and will index the files.

Reply:
@amberdixon am i correct on this?